README.md

```diff
@@ -5,167 +5,145 @@
-kubectl delete -f examples/nginx-app/base.yaml
-```
+1. To check that the nginx deployment and service are gone, run:
 
-## Architecture
+    ```
+    kubectl get deployments --namespace=nginx-example
+    kubectl get services --namespace=nginx-example
+    kubectl get namespace/nginx-example
+    ```
 
-Each of Heptio Ark's operations (Backups, Schedules, and Restores) are custom resources themselves, defined using [CRDs][20]. Their accompanying [custom controllers][21] handle them when they are submitted to the Kubernetes API server.
+    You should get no results.
+    
+    NOTE: You might need to wait for a few minutes for the namespace to be fully cleaned up.
 
-As mentioned before, Ark runs in two different modes:
+### Restore
 
-* **Ark client**: Allows you to query, create, and delete the Ark resources as desired.
+1. Run:
 
-* **Ark server**: Runs all of the Ark controllers. Each controller watches its respective custom resource for API operations, performs validation, and handles the majority of the cloud API logic (e.g. interfacing with object storage and persistent volumes).
+    ```
+    ark restore create nginx-backup
+    ```
 
-Looking at a specific example--an `ark backup create test-backup` command triggers the following operations:
+1. Run:
 
-![19]
+    ```
+    ark restore get
+    ```
 
-1. The *ark client* makes a call to the Kubernetes API server, creating a `Backup` custom resource (which is stored in [etcd][22]).
+    After the restore finishes, the output looks like the following:
 
-2. The `BackupController` sees that a new `Backup` has been created, and validates it.
+    ```
+    NAME                          BACKUP         STATUS      WARNINGS   ERRORS    CREATED                         SELECTOR
+    nginx-backup-20170727200524   nginx-backup   Completed   0          0         2017-07-27 20:05:24 +0000 UTC   <none>
+    ```
 
-3. Once validation passes, the `BackupController` begins the backup process. It collects data by querying the Kubernetes API Server for resources.
+NOTE: The restore can take a few moments to finish. During this time, the `STATUS` column reads `InProgress`.
 
-4. Once the data has been aggregated, the `BackupController` makes a call to the object storage service (e.g. Amazon S3) to upload the backup file.
+After a successful restore, the `STATUS` column is `Completed`, and `WARNINGS` and `ERRORS` are 0. All objects in the `nginx-example` namespacee should be just as they were before you deleted them.
 
-5. By default, Ark also makes disk snapshots of any persistent volumes, using the appropriate cloud service API. (This can be disabled via the option `--snapshot-volumes=false`)
+If there are errors or warnings, you can look at them in detail:
 
-## Extensibility
+```
+ark restore describe <RESTORE_NAME> -o yaml
+```
 
-Ark has multiple mechanisms for extending the core functionality to meet your individual backup/restore needs:
+For more information, see [the debugging information][18].
```

Comment:
"Information" is repeated twice in the same sentence - maybe "see the debugging section" for the link text? Or "For more details, see the debugging information"